Symfony/CS/Tests/Tokenizer/TokensAnalyzerTest.php

```diff
@@ -55,6 +61,8 @@ public function bar5($data)
         $this->assertSame('property', $elements[3]['type']);
         $this->assertSame('method', $elements[4]['type']);
         $this->assertSame('method', $elements[5]['type']);
+
+//        var_dump($elements);
```

Comment:
oops!
fix in https://github.com/SpacePossum/PHP-CS-Fixer/commit/10c80f24c7de5f5ed54cbc7b351534fbbfda1078